db/migrations/schema-objects/20230324145608.do.hdfstatus-trigger.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TRIGGER [mtc_admin].[hdfStatusUpdatedAtTrigger]
```

Comment:
This seems overkill to me as the values in the lookup table can only be inserted via migrations - there aren't any dynamic inserts that I know of.

Reply:
it's just for consistency - other lookup tables have the trigger.